src/app/data-planes/components/data-plane-traffic/ServiceTrafficGroup.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="service-traffic-group">
+    <KCard
+      :class="`type-${props.type}`"
+    >
+      <template #body>
+        <slot name="default" />
+      </template>
+    </KCard>
+  </div>
+</template>
+<script lang="ts" setup>
+const props = defineProps<{
+  type: string
+}>()
+</script>
+<style lang="scss" scoped>
+.service-traffic-group :deep(.k-card-body) {
+  display: flex;
+  flex-direction: column;
+  gap: $kui-space-40;
+}
```

Comment:
**Suggestion**: Instead of overriding `.k-card-body`, I suggest adding a wrapping element around `<slot name="default" />` which these styles are attached to. This way we don’t depend on class names out of our control.

As an aside, for `<slot name="default" />`, you can always omit `name="default"` as it’s the default name.

Reply:
I removed all the places where I was using `.kong` CSS selectors or similar.

As for the  `<slot name="default" />` I don't think it matters either way, most of me thinks being explicit is good for when/if more slots get added but 🤷 . I didn't change it.